doc/windows_compute/NBL_processing.md

```diff
@@ -0,0 +1,35 @@
+# NET_BUFFER_LIST processing
+
+## NET_BUFFER_LIST and vr_packet relationship
+
+![nbl-vrpacket](nbl-fragmenting.png)
+
+- `NET_BUFFER_LIST` is a linked list of packet containers
+- `NET_BUFFER` represents a single packet
+- `MDL` linked list represent a series of memory buffers
+- Windows API mostly accepts `NET_BUFFER_LIST` structures
+- vRouter expects `vr_packet` to be a single network packet
+- Thus Windows vRouter code must assure that `vr_packet` will encapsulate:
+    - single `NET_BUFFER_LIST`
+    - single `NET_BUFFER`
+
+## Receiving fragmented packets from container
+
+![nbl-fragmented-container-input](nbl-fragmentation1.png)
+
+**Scenario**: Packet is fragmented in container, before reaching vRouter
+
+- Extension receives a single `NET_BUFFER_LIST` with a set of `NET_BUFFER` structures
+- Each `NET_BUFFER` must be in its own `NET_BUFFER_LIST`
```

Comment:
> must

When reading this doc, this "must" is a little bit out of context. Please restate that this relationship is necessary to fulfill dp-core-NBL relationship assumptions.